api-common/src/main/java/org/openmrs/module/metadatasharing/metadata/MetadataPackageConfig.java

```diff
@@ -0,0 +1,69 @@
+/**
+ * The contents of this file are subject to the OpenMRS Public License
+ * Version 1.0 (the "License"); you may not use this file except in

```

Comment:
This is our current license: https://github.com/openmrs/openmrs-core/blob/master/api/src/main/java/org/openmrs/Allergy.java#L1-L9

Reply:
alryt. let correct it